spec/jobs/event_notification_job_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe EventNotificationJob do
+  subject(:job) { described_class }
+  let(:events_requiring_notification) { Event.requiring_notification }
+  context 'when there are several events requiring notification' do
+    let!(:required_notif_1) do
+      FactoryGirl.create(
+        :event, date: Date.current - Event::NOTIFICATION_PERIOD
+      )
+    end
+    let!(:required_notif_2) do
+      FactoryGirl.create(
+        :event, date: Date.current - Event::NOTIFICATION_PERIOD
+      )
+    end
+    context 'when those events have presentations' do
+      before do
+        events_requiring_notification.each do |event|
+          FactoryGirl.create(:presentation, event: event)
+        end
+      end
+      it 'sends an email to each presenter' do
+        expect { job.perform }
+          .to change { ActionMailer::Base.deliveries.count }
+          .by(events_requiring_notification.length)
+      end
+    end
+  end
+  context 'when there are no events requiring notification' do
+    let!(:unrequired_notif) do
+      FactoryGirl.create(
+        :event, date: Date.current + Event::NOTIFICATION_PERIOD
+      )
+    end
+    it 'sends zero emails' do
+      expect(events_requiring_notification.length).to eq(0)
+      expect { job.perform }
```

Comment:
described class.perform should probably be your subject since thats what you are "expecteing" in both tests